pyowm.exceptions package
========================

Submodules
----------

pyowm.exceptions.api_call_error module
--------------------------------------

.. automodule:: pyowm.exceptions.api_call_error
    :members:
    :undoc-members:
    :show-inheritance:

pyowm.exceptions.api_response_error module
------------------------------------------

.. automodule:: pyowm.exceptions.api_response_error
    :members:
    :undoc-members:
    :show-inheritance:

pyowm.exceptions.not_found_error module
---------------------------------------

.. automodule:: pyowm.exceptions.not_found_error
    :members:
    :undoc-members:
    :show-inheritance:

pyowm.exceptions.parse_response_error module
--------------------------------------------

.. automodule:: pyowm.exceptions.parse_response_error
    :members:
    :undoc-members:
    :show-inheritance:


pyowm.exceptions.unauthorized_error module
------------------------------------------

.. automodule:: pyowm.exceptions.unauthorized_error
    :members:
    :undoc-members:
    :show-inheritance:

Module contents
---------------

.. automodule:: pyowm.exceptions
    :members:
    :undoc-members:
    :show-inheritance:
